### ISSUE TITLE
x99 f8 ver.2.0

### ISSUE THREAD

**Issue by solnyshok** [Koshak1013/HuananzhiX99_BIOS_mods#17]
пытался шить рекомендованную для x99f8 buos 2020h05-25: v007, v006, стандартную. так как после первой прошивки с fptw плата перестает запускаться с ошибкой bf (или 6f), шью прищепкой программатором. ваши биос почему-то не стартуют. заводская прошивка с сайта huananzhi всегда оживляет материнку. что я делаю не так?

**Comment by solnyshok** [comment]
может ли причина быть в том что у меня память DDR4 2666 и cpu-z показывает что у нее все spd-профили на 1333MHz. (но hwinfo показывает разные скорости от 600 до 1333). на стандартном биосе память работает на 2100MHz, а вот модифицированный биос похоже пытается ее использовать на 2666 и стабильно висит

**Comment by Koshak1013** [comment]
Даже не знаю в чём проблема, модификации в биосе никак не связаны с частотой памяти. Зав.биос с сайта тоже майский?

**Comment by solnyshok** [comment]
заводской биос с сайта huananzhi теперь 2021-01-20

**Comment by Koshak1013** [comment]
Январский, он как бы на базе августовского. В общем совсем другой, после мая поменяли основу. Но май всегда работал, и считается лучшим, с наименьшим кол-вом проблем. Если есть желание, попробуйте ещё раз прошить майский, предварительно изменив в нём стандартную частоту памяти в AMIBCP на рабочую провенную.

**Comment by solnyshok** [comment]
![image](https://user-images.githubusercontent.com/13075980/116079791-e9a7d280-a6a0-11eb-9f9f-fa658e0c3013.png)
в AMIBCP нашел такое -- отсутствует пункт Memory Frequency = Autro.  может ли это быть причиной?

**Comment by Koshak1013** [comment]
Так и должно быть, настройка частоты в меню Memory Configuration

**Comment by Koshak1013** [comment]
Да, если используется подходящий процессор. У некоторых моделей процессоров максимальная поддерживаемая частота 2133 или даже 1866.
